tests/validation/tests/v3_api/test_logging.py

```diff
@@ -0,0 +1,215 @@
+import pytest
+import base64
+import time
+from .common import random_test_name
+from .common import get_admin_client_and_cluster
+from .common import create_project_and_ns
+from .common import get_project_client_for_token
+from .common import create_kubeconfig
+from .common import get_admin_client
+from .common import wait_for_app_to_active
+from .common import ADMIN_TOKEN
+
+namespace = {"p_client": None, "ns": None, "cluster": None, "project": None,
+             "name_prefix": None, "admin_client": None, "sys_p_client": None}
+
+fluentd_app_name = "rancher-logging"
+fluentd_secret_name = "rancher-logging-fluentd"
+endpoint_host = "rancher.com"
+endpoint_port = "24224"
+username = "user1"
+password = "my_password"
+shared_key = "my_shared_key"
+weight = 100
+
+
+def test_fluentd_target(request):
+    cluster = namespace["cluster"]
+    cluster_logging = create_cluster_logging(fluentd_target_without_ssl())
+    request.addfinalizer(lambda: delete_cluster_logging(cluster_logging))
```

Comment:
We can use remove_resource from conftest.py

Reply:
Hi @mrajashree remove_resource is under integration test, these test cases are under validation test, is this function available in validation test?